fabric-blockrenderlayer-v1/src/main/java/net/fabricmc/fabric/api/blockrenderlayer/v1/BlockRenderLayerMap.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.blockrenderlayer.v1;
+
+import net.fabricmc.fabric.impl.blockrenderlayer.v1.BlockRenderLayerMapImpl;
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockRenderLayer;
+import net.minecraft.fluid.Fluid;
+
+/**
+ * Use to associate blocks or fluids with block render layer other than default.  
+ * Replaces the {@code renderLayer} property previously on {@code Block}.<p>
```

Comment:
Adding some more description would be nice, not everyone knows what a "render layer" is supposed to be and what is going on with this "default". The historical explanation is good, but only useful for experienced users.